webapp/sass/components/_suggestion-list.scss

```diff
@@ -44,3 +44,33 @@
 .suggestion-list--bottom {
     position: relative;
 }
+
+.suggestion-list__divider {
+    line-height: 21px;
+    margin: 5px 0px 0px 5px;
+    position: relative;
+
+    &:first-child {
+        margin-top: 5px;
+    }
+
+    > span {
+        color: rgba(51,51,51,0.7);
+        background: #f2f4f8;
```

Comment:
I think these colors should be specified in theme-independent way but I don't know how to do that. I'll update it if you let me know.